x-pack/plugins/observability_ai_assistant/public/components/prompt_editor/prompt_editor_natural_language.tsx

```diff
@@ -54,12 +55,18 @@ export function PromptEditorNaturalLanguage({
     if (textarea) {
       textarea.focus();
     }
-  }, [handleResizeTextArea]);
+  }, []);
 
   useEffect(() => {
     handleResizeTextArea();
   }, [handleResizeTextArea]);
 
+  useEffect(() => {
+    if (prompt === undefined) {
+      handleResizeTextArea();
+    }
+  }, [handleResizeTextArea, prompt]);
+
```

Comment:
Do we need both `useEffect` calls? 

```suggestion

  useEffect(() => {
    if (prompt === undefined) {
      handleResizeTextArea();
    }
  }, [handleResizeTextArea, prompt]);

```